helm/templates/certificate_webhook-conf.yaml

```diff
@@ -13,19 +13,98 @@ metadata:
     {{- include "helm.labels" . | nindent 4 }}
 type: Opaque
 data:
-  tls.crt: {{ $certs.Cert | b64enc }}
-  tls.key: {{ $certs.Key | b64enc }}
+  tls.crt: {{ default ($certs.Cert | b64enc) (include "getInstalledTLSCert" .) }}
+  tls.key: {{ default ($certs.Key | b64enc) (include "getInstalledTLSKey" .) }}
 ---
```

Comment:
the tls-certificate and the webhook need the same certificate. otherwise conny will not work. if we want the use helm for generating said certificate, both of these (tls and webhook) need to be defined in the same file so they can access the same certificate. otherwise helm would generate 2 different certificates and thus break conny. the name comes from the combination of the two resources that are defined in there. the tls certificate and the webhook.